packages/connect-examples/react-native-demo/deep-link/README.md

```
@@ -0,0 +1,44 @@
+# Deep Link Playground
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)